parsers/FR_O.py

```diff
@@ -185,21 +182,25 @@ def fetch_data(
     if not isinstance(data, list):
         raise ParserException(
             "FR_O.py",
-            f"Unexpected data format for {zone_key} for {target_datetime.strftime('%Y')}"
+            f"Unexpected data format for {zone_key} for {target_datetime}"
             if target_datetime
             else f"Unexpected data format for {zone_key}.",
             zone_key,
         )
-    return data, DATE_STRING_MAPPING[zone_key]
+    source = url.split("//")[1].split("/")[0]
```

Comment:
Do we want to just set the source as EDF for all of these datatypes and regions?
All our data comes from EDF subdomains.

Reply:
i think edf is good enough

Reply:
today we use opendata.edf.fr, i think we should keep this source

Reply:
We use different ones of all the regions, FR-COR for example use opendata-corse.edf.fr

So I would prefer using edf.fr as the source as it covers them all and is the organisation in it self.

Reply:
i would use opendata.edf.fr to signify where we get the data too. Edf is a massive company and it can be hard to get back to the data source

Reply:
True, let's do `opendata.edf.fr` then 🙂